templates/vue/src/utils/Helpers.js

```diff
@@ -143,9 +143,16 @@ export default class Helpers {
       return true
     }
 
-    // Check 2: User is the author of the node
-    if (node.author && wpData.currentUser.ID == parseInt(node.author.id)) {
-      return true
+    // Check 2: User is the author of the node, if the node is a draft
+    if (
+      node.author &&
+      wpData.currentUser.ID == parseInt(node.author.id) &&
+      node.status === "draft"
+    ) {
```

Comment:
You are erroneously removing the ability for node authors to edit their own node here (if their node is not draft). Published nodes should remain editable by their authors (unless they were submitted and accepted).